README.md

```diff
@@ -4,10 +4,29 @@
 
 **`@bagaar/ember-permissions` is built and maintained by [Bagaar](http://bagaar.be).**
 
-[![Build Status](https://travis-ci.org/Bagaar/ember-permissions.svg?branch=master)](https://travis-ci.org/Bagaar/ember-permissions)
+[![Build Status](https://travis-ci.org/Bagaar/ember-permissions.svg?branch=master)](https://travis-ci.org/Bagaar/ember-permissions) [![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg)](https://conventionalcommits.org)
```

Comment:
My first thought when I saw this badge is that it contained the version of the addon, but it's the version of conventional commits. Does a variant without the version exist? Another option is to add badge with the current addon version before the conventional commit badge.